internal/cmd/cmd_zcount_watch.go

```diff
@@ -69,7 +69,7 @@ func evalZCOUNTWATCH(c *Cmd, s *dstore.Store) (*CmdRes, error) {
 }
 
 func executeZCOUNTWATCH(c *Cmd, sm *shardmanager.ShardManager) (*CmdRes, error) {
-	if len(c.C.Args) == 0 {
```

Comment:
`==0` is correct. For `.watch` commands the check on arguments is handled at primary command level. This way, we need not make changes at two places when we support other args. Revert this change.